clarity_ext/dilution.py

```diff
@@ -220,9 +230,9 @@ def calculate_transfer_volumes(self):
                     transfer.buffer_volume *= scale_factor
                     transfer.scaled_up = True
             except (TypeError, ZeroDivisionError) as e:
-                transfer.sample_volume = None
-                transfer.buffer_volume = None
-                transfer.has_to_evaporate = None
+                transfer.sample_volume = 0
```

Comment:
And then it is up to the script writer in `clarity-snpseq` to handle it in a sensible way?